devops/developer-api/openapi.yml

```diff
@@ -0,0 +1,1843 @@
+        db_max_clients:
+          type: integer
+          description: Max number of concurrent clients can be opened on this database currently
+          format: int64
+          example: 1000
+        db_max_request_size:
+          type: integer
+          description: Max size of a request that will be accepted by the database currently(in bytes)
+          format: int64
+          example: 1048576
+        db_disk_threshold:
+          type: integer
+          description: Total disk size limit that can be used for the database currently(in bytes)
+          format: int64
+          example: 268435456
+        db_max_entry_size:
+          type: integer
+          description: Max size of an entry that will be accepted by the database currently(in bytes)
+          format: int64
+          example: 104857600
+        db_memory_threshold:
+          type: integer
+          description: Max size of a memory the database can use(in bytes)
+          format: int64
+          example: 268435456
+        db_daily_bandwidth_limit:
+          type: integer
+          description: Max daily bandwidth can be used by the database(in bytes)
+          format: int64
+          example: 53687091200
+        db_max_commands_per_second:
+          type: integer
+          description: Max number of commands can be sent to the database per second
+          format: int64
+          example: 1000
+        db_request_limit:
+          type: integer
+          description: Total number of commands can be sent to the database
+          format: int64
+          example: 10000
+        cluster_id:
+          type: string
+          description: Cluster ID of the database
+        pool_id:
+          type: string
+          description: Pool ID of the database
+        type:
+          type: string
+          description: Payment plan of the database
+          example: paid
+        budget:
+          type: integer
+          description: Allocated monthly budget for the database
+          format: int64
+        primary_region:
+          type: string
+          description: Primary region of the database cluster
+        primary_members:
+          type: array
+          items:
+            type: string
+          description: List of primary regions in the database cluster
+        all_members:
+          type: array
+          items:
+            type: string
+          description: List of all regions in the database cluster
+        eviction:
+          type: boolean
+          description: Entry eviction is enabled
+        auto_upgrade:
+          type: boolean
+          description: Automatic upgrade capability is enabled
+        consistent:
+          type: boolean
+          description: Strong consistency mode is enabled
+        reserved_per_region_price:
+          type: integer
+          description: Reserved price per region for enterprise plans
+          format: int64
```

Comment:
consistent isn't returned. reserved_per_region_price is returned but I don't know what it is. Asked [here](https://upstash.slack.com/archives/C099375LDKM/p1756715826337479)